src/main/java/com/apperian/api/signing/SignApplicationResponse.java

```diff
@@ -1,9 +1,8 @@
 package com.apperian.api.signing;
 
-import com.apperian.api.ApperianResponse;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class SignApplicationResponse extends ApperianResponse {
+public class SignApplicationResponse {
     @JsonProperty("signing_status")
```

Comment:
Is there a reason this needs to be a @JsonProperty annotations here?  None of the other response objects have them.

Reply:
As for JSON properties we tend to use underscore notation and for Java variables camelcase one, these JsonProperty are useful to map the names mismatch. Indeed it is probably a good practise to have them always, even when they have the same name, so that any refactor renaming variables do not break the integration. Anyway I will review the code and remove them when they are not needed (i.e. when the name in Java is the same name as the field in JSON).